src/js/paste.js

```diff
@@ -129,15 +145,15 @@ var PasteHandler;
                 } else {
                     html = Util.htmlEntities(pastedPlain);
                 }
-                Util.insertHTMLCommand(this.options.ownerDocument, html);
+                Util.insertHTMLCommand(this.ownerDocument, html);
             }
         },
 
         cleanPaste: function (text) {
             var i, elList, workEl,
-                el = Selection.getSelectionElement(this.options.contentWindow),
+                el = Selection.getSelectionElement(this.contentWindow),
```

Comment:
unless the defaultsMixer thing is passing contentWindow and friends as ctor props, this is the wrong reference I think? not that I am _opposed_ to Extension's getting a default .document and .window property relayed from base options ... so instead of `Util.insertHTML(this.base.option.ownerDocuemnt, html)` you'd `Util.inserHTML(this.document, html)`